src/nebula/Queue/Implementation/RedisJobQueue.cs

```diff
@@ -15,8 +15,11 @@ public class RedisJobQueue<TItem> : IJobQueue<TItem> where TItem : IJobStep
         [ComponentPlug]
         public IRedisManager RedisManager { get; set; }
 
+        public bool QueueExistenceChecked { get; set; }
```

Comment:
Why did you add this? Is it only for Test purpose?
If so, I don't think this is the best way to go. It doesn't need to be in the original RegisJobQueue implementation. Maybe a subclass?